### PR TITLE
Finalized Fixes for the new website! 

### PR DESCRIPTION
**Pull request by jaredwhanisch**
Joe left some comments for me to fix on the [beta version](https://staging-mst-ka.web.app/) of the website, this PR is a response to those comments.

You can see the (hopefully) final version of the website [here](https://mst-ka--final-fixes-fj64t23w.web.app).

### CODE REVIEW

File: pages/about/academics.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+              understands that success is fostered through hard work,
+              discipline, and a drive to be the very best.
+              <br />
+              <br />
+              With that hard work we have received the top GPA on campus for the
+              last 5 semesters in a row as of Spring 2022. If you&apos;d like
+              see how our grades compare among other organizations on campus,
+              please{" "}
+              <Link href="https://involvement.mst.edu/media/studentsupport/involvement/documents/KA.pdf#220802105839">
+                click here
```
Comment (joe-studer-18) [inline pages/about/academics.js:42]: Bit of a nit - should we iframe or embed this directly to our site? Worst case, can you update the link to [open in a new window](https://www.freecodecamp.org/news/how-to-use-html-to-open-link-in-new-tab/)?

File: pages/alumni/index.js [hunk 2]
```diff
@@ -28,31 +28,52 @@ const newsletters = [
...
+            Alumni Reporting Form!
+          </Link> 
+          
+        </Typography>
+
+        <Grid container spacing={6} sx={{marginTop: "0rem"}}>
           {newsletters.map((newsletter, index) => (
-            <Grid key={index} item mobile={12} tablet={4} laptop={3}>
+            <Grid key={index} item mobile={12} tablet={6}>
               <Newsletter
```
Comment (joe-studer-18) [inline pages/alumni/index.js:77]: Similar to the other links - can you update the ones for this component to open in a new tab?

**Review by joe-studer-18 — APPROVED** [review]
Couple Small changes that don't need to happen now, but should probably follow shortly after

- awards/national-awards
This section should probably mirror campus awards (which really looks soooo much better) by using a table view - either grouped by award, or grouped by year. This bit would need a bit of help from the 5 since nationals give a lot of awards each year, but it'd really help to hammer home with the "heres how much we win" story.

- awards/scholarships
This probably makes more sense in "About"?

> 

>

File: pages/about/academics.js (hunk 1 above)
Comment (jaredwhanisch) [reply pages/about/academics.js:42]: Good thinking, tried the iframe for the pdf, kinda looks bad, so I opted for opening the link in a new tab.

**Comment by jaredwhanisch** [comment]
Yeah I'll make an issue for national awards, good idea moving the scholarships page, that makes way more sense now that you point it out. I also noticed when looking on my phone, that there was an issue with the footer covering some content so I fixed that. I deployed the changes to a hosting channel that you can see here: https://mst-ka--final-fixes-fj64t23w.web.app/

File: pages/alumni/index.js (hunk 2 above)
Comment (jaredwhanisch) [reply pages/alumni/index.js:77]: Opening newsletters in a new tab now.

**Pull request merged**